10:17:55.481 DEBUG - FiltreSession doFilter - UCS indique que l'utilisateur n'est pas connecté 
10:17:55.481 DEBUG - FiltreSession Requête pour page non login (http://localhost:8383/Quartis-Web/moduleChange.ajx?moduleID=OPTIMA_Controle_Process&userID=administrateur) mais pas de user en session. 
10:17:55.497 DEBUG - FiltreSession doFilter - UCS indique que l'utilisateur n'est pas connecté 
10:17:55.519 DEBUG - FiltreSession doFilter - UCS indique que l'utilisateur n'est pas connecté 
10:17:55.520 DEBUG - FiltreSession Requête pour page non login (http://localhost:8383/Quartis-Web/xxx.cas?action=deconnexion&userID=administrateur) mais pas de user en session. 
10:17:55.522 DEBUG - FiltreSession doFilter - UCS indique que l'utilisateur n'est pas connecté 
10:17:55.522 DEBUG - FiltreSession doFilter - UCS indique que l'utilisateur n'est pas connecté 
10:17:58.245 DEBUG - FiltreSession doFilter - UCS indique que l'utilisateur n'est pas connecté 
10:17:58.262 DEBUG - LoginManagedBean user.id = administrateur, connecté par login/mdp Osys 
10:17:58.692 ERROR - FiltreSession Erreur relevée par le filtre général : org.apache.jasper.JasperException: org.apache.jasper.JasperException: javax.servlet.ServletException: quartisspv.app.ApplicationException: Preference level1 parametre : format XML invalide. org.apache.jasper.JasperException: org.apache.jasper.JasperException: org.apache.jasper.JasperException: javax.servlet.ServletException: quartisspv.app.ApplicationException: Preference level1 parametre : format XML invalide.
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:491)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:401)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:313)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:260)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:64)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:646)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:436)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:374)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:302)
	at sun.reflect.GeneratedMethodAccessor304.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.bull.javamelody.JspWrapper.invoke(JspWrapper.java:150)
	at net.bull.javamelody.JdbcWrapper$DelegatingInvocationHandler.invoke(JdbcWrapper.java:286)
	at com.sun.proxy.$Proxy119.forward(Unknown Source)
	at quartisspv.action.level1.Level1Action.processPerformed(Level1Action.java:39)
	at quartisspv.action.JspAction.actionPerformed(JspAction.java:31)
	at quartisspv.action.AbstractAction.run(AbstractAction.java:94)
	at quartisspv.servlet.ActionServlet.processRequest(ActionServlet.java:76)
	at quartisspv.servlet.ActionServlet.doGet(ActionServlet.java:103)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:617)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:201)
	at net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:178)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at quartisweb.servlet.EncodingFilter.doFilter(EncodingFilter.java:26)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:151)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.jasper.JasperException: org.apache.jasper.JasperException: javax.servlet.ServletException: quartisspv.app.ApplicationException: Preference level1 parametre : format XML invalide.
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:491)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:401)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:313)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:260)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:64)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:646)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:551)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:488)
	at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.bull.javamelody.JspWrapper.invoke(JspWrapper.java:150)
	at net.bull.javamelody.JdbcWrapper$DelegatingInvocationHandler.invoke(JdbcWrapper.java:286)
	at com.sun.proxy.$Proxy119.include(Unknown Source)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:968)
	at org.apache.jsp.sup.level1_jsp._jspService(level1_jsp.java:154)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:377)
	... 52 more
Caused by: org.apache.jasper.JasperException: javax.servlet.ServletException: quartisspv.app.ApplicationException: Preference level1 parametre : format XML invalide.
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:491)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:401)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:313)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:260)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:64)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:646)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:551)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:488)
	at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.bull.javamelody.JspWrapper.invoke(JspWrapper.java:150)
	at net.bull.javamelody.JdbcWrapper$DelegatingInvocationHandler.invoke(JdbcWrapper.java:286)
	at com.sun.proxy.$Proxy119.include(Unknown Source)
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:968)
	at org.apache.jsp.sup.preference_jsp._jspService(preference_jsp.java:86)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:377)
	... 74 more
Caused by: javax.servlet.ServletException: quartisspv.app.ApplicationException: Preference level1 parametre : format XML invalide.
	at org.apache.jasper.runtime.PageContextImpl.doHandlePageException(PageContextImpl.java:862)
	at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:791)
	at org.apache.jsp.sup.pref_002dtab_002dparams_jsp._jspService(pref_002dtab_002dparams_jsp.java:489)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:377)
	... 96 more
Caused by: quartisspv.app.ApplicationException: Preference level1 parametre : format XML invalide.
	at quartisspv.action.level1.Level1PreferenceDelegate.getParameters(Level1PreferenceDelegate.java:81)
	at org.apache.jsp.sup.pref_002dtab_002dparams_jsp._jspService(pref_002dtab_002dparams_jsp.java:87)
	... 99 more
Caused by: javax.xml.bind.UnmarshalException
 - with linked exception:
[org.xml.sax.SAXParseException; systemId: file:/C:/Osys/QUARTIS_PRO/quartis-pro-web/users/administrateur/level1.xml; lineNumber: 1; columnNumber: 579; Content is not allowed in trailing section.]
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.createUnmarshalException(Unknown Source)
	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.createUnmarshalException(Unknown Source)
	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(Unknown Source)
	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(Unknown Source)
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(Unknown Source)
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(Unknown Source)
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(Unknown Source)
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(Unknown Source)
	at quartisspv.action.level1.Level1PreferenceDelegate.getParameters(Level1PreferenceDelegate.java:46)
	... 100 more
Caused by: org.xml.sax.SAXParseException; systemId: file:/C:/Osys/QUARTIS_PRO/quartis-pro-web/users/administrateur/level1.xml; lineNumber: 1; columnNumber: 579; Content is not allowed in trailing section.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source)
	at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
	at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentScannerImpl$TrailingMiscDispatcher.dispatch(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	... 107 more

10:18:02.160 DEBUG - TableDivTransformer createFileName ()--> Nom du fichier export = _Controle_Gammes_EnCours_Liste 
10:18:02.160 DEBUG - TableDivTransformer createFileName ()--> Nom du fichier export = _Controle_Gammes_EnCours_Liste 
10:18:02.160 DEBUG - TableExcelTransformer actionPerformed - Ecriture dans le fichier 'C:\Osys\QuartisTools\Tomcat\webapps\Quartis-Web\.\work\_0.0.0.0.0.0.0.1-administrateur\_Controle_Gammes_EnCours_Liste\_Controle_Gammes_EnCours_Liste.xls'... 
10:18:02.160 DEBUG - TableExcelTransformer actionPerformed - Ecriture dans le fichier 'C:\Osys\QuartisTools\Tomcat\webapps\Quartis-Web\.\work\_0.0.0.0.0.0.0.1-administrateur\_Controle_Gammes_EnCours_Liste\_Controle_Gammes_EnCours_Liste.xls'... 
10:18:02.160 DEBUG - TableExcelTransformer actionPerformed - Ecriture dans le fichier 'C:\Osys\QuartisTools\Tomcat\webapps\Quartis-Web\.\work\_0.0.0.0.0.0.0.1-administrateur\_Controle_Gammes_EnCours_Liste\_Controle_Gammes_EnCours_Liste.xls'... 
10:18:02.160 DEBUG - TableExcelTransformer actionPerformed - Ecriture dans le fichier 'C:\Osys\QuartisTools\Tomcat\webapps\Quartis-Web\.\work\_0.0.0.0.0.0.0.1-administrateur\_Controle_Gammes_EnCours_Liste\_Controle_Gammes_EnCours_Liste.xls'... 
10:18:02.173 DEBUG - TableExcelTransformer actionPerformed - Fin de l'écriture dans le fichier. 
10:18:02.173 DEBUG - TableExcelTransformer actionPerformed - Fin de l'écriture dans le fichier. 
10:18:06.085 DEBUG - GSqlQueryGeneratorSqlServer --->table =CONTROLE_LISTE query =set rowcount 3000  SELECT CODE_ARTI  , CODE_CONT  , dbo.OPTIMA_TRADUCTION(DESI_CONT,'fr') +'@KEYLANG_'+DESI_CONT AS DESI_CONT  , CODE_SITE  FROM CONTROLE_LISTE  WHERE ( CODE_SITE = 'MAR')  ORDER BY CODE_ARTI ASC  
10:18:09.043 DEBUG - GSqlQueryGeneratorSqlServer --->table =CONTROLE_MESURE query =set rowcount 3000  SELECT CODE_ARTI  , CODE_CONT  , CODE_MESU  , NUME_ORDR  , CODE_APPA  , NOMB_MESU  , CODE_TYPE_SAIS  , CODE_TYPE_LIST  , VALE_MINI  , VALE_MAXI  , VALE_CIBL  , COEF_FREQ  , FLAG_DEBU_PROD  , dbo.OPTIMA_TRADUCTION(QUES,'fr') +'@KEYLANG_'+QUES AS QUES  , dbo.OPTIMA_TRADUCTION(COMM,'fr') +'@KEYLANG_'+COMM AS COMM  , CODE_SITE  FROM CONTROLE_MESURE  WHERE ( CODE_SITE = 'MAR')  ORDER BY CODE_ARTI ASC  
15:51:05.500  INFO - PropertiesDelegate Destruction du properties manager ! 
15:54:26.981 DEBUG - TomcatListener contextInitialized - servletContext.realPath : C:\Osys\QuartisTools\Tomcat\webapps\Quartis-Web 
15:54:27.028  INFO - PropertiesDelegate Ouverture du fichier de configuration à l'emplacement : C:\Osys\QuartisTools\Tomcat/conf/quartis-web.properties 
15:54:27.122  INFO - TomcatListener -----> Supression des fichier temporaires 
15:54:27.730  INFO - TomcatListener Ouverture du fichier indiquant la base de données : C:\Osys\QuartisTools\Tomcat/conf/datasources.xml... 
15:54:27.948  INFO - DataSourceManager Création de la datasource : QuartisWeb-PU 
15:54:29.087 ERROR - GSql  org.apache.commons.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (Network error IOException: Connection refused: connect)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1225)
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:880)
	at quartisweb.bd.AccesBD.getConnection(AccesBD.java:140)
	at quartisweb.generic.sql.GSql.getListCodeSites(GSql.java:923)
	at quartisweb.id.global.UserGlobalManager.verifySiteTiret(UserGlobalManager.java:124)
	at quartisweb.id.global.UserGlobalManager.creerListUser(UserGlobalManager.java:85)
	at quartisweb.id.global.UserGlobalManager.init(UserGlobalManager.java:35)
	at quartisweb.tomcat.TomcatListener.contextInitialized(TomcatListener.java:109)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4135)
	at org.apache.catalina.core.StandardContext.start(StandardContext.java:4630)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:791)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:771)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:546)
	at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:637)
	at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:563)
	at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:498)
	at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1277)
	at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:321)
	at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:119)
	at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1053)
	at org.apache.catalina.core.StandardHost.start(StandardHost.java:785)
	at org.apache.catalina.core.ContainerBase.start(ContainerBase.java:1045)
	at org.apache.catalina.core.StandardEngine.start(StandardEngine.java:445)
	at org.apache.catalina.core.StandardService.start(StandardService.java:519)
	at org.apache.catalina.core.StandardServer.start(StandardServer.java:710)
	at org.apache.catalina.startup.Catalina.start(Catalina.java:581)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:289)
	at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:414)
Caused by: java.sql.SQLException: Network error IOException: Connection refused: connect
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:385)
	at net.sourceforge.jtds.jdbc.ConnectionJDBC3.<init>(ConnectionJDBC3.java:50)
	at net.sourceforge.jtds.jdbc.Driver.connect(Driver.java:182)
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:294)
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1247)
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1221)
	... 31 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.sourceforge.jtds.jdbc.SharedSocket.createSocketForJDBC3(SharedSocket.java:304)
	at net.sourceforge.jtds.jdbc.SharedSocket.<init>(SharedSocket.java:255)
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:310)
	... 37 more

